README.rst

```diff
@@ -22,13 +25,13 @@ OpenWISP 2 monitoring module (Work in progress).
 Available Features
 ------------------
 
-* Displays ``Uptime``, ``Memory Status``, ``Load Status``, ``Addresses``, ``Neighbours`` of the **Device**
-* Displays ``Monitoring Graphs`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of lively charts
+* Displays `Uptime, Memory Status, Load Status, Addresses, Neighbours of the Device <docs/device_status.png>`_
```

Comment:
absolute paths

Reply:
Memory and Load Status is not very meaningful. Please reword, e.g. memory usage and CPU load and mention `disk usage` too

Reply:
Currently, the `Device Status` is not displaying `disk_usage`, it can be changed easily but do we need to change it (might be redundant since the graph is already doing it.)